src/avram/model.cr

```diff
@@ -44,7 +44,7 @@ class Avram::Model
     end
   end
 
-  def_equals @id
+  def_equals id
```

Comment:
Use the method since the instance var is not defined if using a different primary key name